src/main/kotlin/com/hibob/feedback/resource/ResponseResource.kt

```diff
@@ -0,0 +1,29 @@
+package com.hibob.feedback.resource
+
+
+import com.hibob.feedback.dao.ResponseInput
+import com.hibob.feedback.service.ResponseService
+import jakarta.ws.rs.*
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Component
+
+@Component
+@Path("/api/response")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class ResponseResource(private val responseService: ResponseService, private val feedbackResource: FeedbackResource) {
```

Comment:
instead of injecting the feedbackResource here, let's move the throwIfNotAuthorized and getActiveUserOrThrow functions to a helper class